hw/top_earlgrey/ip/clkmgr/data/autogen/clkmgr.hjson

```diff
@@ -154,6 +154,11 @@
     },
   ],
 
+  countermeasures: [
+    { name: "BUS.INTEGRITY",
+      desc: "End-to-end bus integrity scheme."
+    }
+  ]
```

Comment:
I am sure I do not know how this works, but why are edits made to a file in the "autogen" directory. Same comment with other files.

Reply:
Some IPs are generated IPs (especially the infrastructural ones like clkmgr, alert handler, interrupt controller, since they depend on top-level configuration). Therefore, their Hjson is templated (extension: `hjson.tpl`), and the generated Hjson files are put under` autogen`.

Reply:
got it, thanks!